src/components/icon/icon.js

```diff
@@ -102,11 +104,13 @@ class Icon extends Component {
    * Generates the encoded svg url for a certain svg path
    *
    * @param {string} path - svg path
+   * @param {?number} width - svg width
+   * @param {?number} height - svg height
    * @returns {string} - encoded svg url
    * @memberof Icon
    */
-  getSVGUrl = (path: string): string => {
-    const svg = `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 1024 1024" width="36" height="36">${path}</svg>`;
+  getSVGUrl = (path: string, width: ?number = 36, height: ?number = 36): string => {
```

Comment:
should be
`width?: number = 36, height?: number = 36`

Reply:
should be w/o "?" Actually . Fixed

Reply:
done